src/dlgTriggerEditor.cpp

```diff
@@ -4613,7 +4655,7 @@ void dlgTriggerEditor::slot_var_selected(QTreeWidgetItem *pItem)
     if (varType == LUA_TTABLE || varType == LUA_TFUNCTION)
     {
         // Edbee doesn't have a readonly option, so I'm using setEnabled
-        mpSourceEditorEdbee->setEnabled(false);
+        // mpSourceEditorEdbee->setEnabled(false);
```

Comment:
🕦 OK I was slow to get my approval in - Vadim's override was okay in this case... 😉